client/src/components/Nav.js

```diff
@@ -115,16 +124,16 @@ class Nav extends React.Component {
       <div className="right__nav">
         <span onClick={this.handleLogout}>Logout</span>
       </div>
-    )
+    );
 
     return (
-    <header>
-      <nav className="landing__nav">
+      <header>
+        <nav className="landing__nav">
           <div className="left__nav">LOGO</div>
           {authed ? isAuth : isNotAuth}
-      </nav>
+        </nav>
 
-              {/* Signup Modal */}
+        {/* Signup Modal */}
 
         <Modal
```

Comment:
I've been struggling with getting the forms to look and act the way I want, but I really like this reactstrap code - looks like it might solve my problems.